tests/test_basic/test_basic.py

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env python
 
+import os
 import sys
-sys.path.insert(0, '..')
+sys.path.insert(0, os.pardir)
+sys.path.insert(0, os.path.join(os.pardir, os.pardir))
```

Comment:
Whoops, testing_harness imports input_set before modifying the path.  If we just move the `sys.path.insert(...)` up one line in testing_harness then we don't need the `../../` in each test.  That was a silly error.